earth_enterprise/src/fusion/portableglobe/gerewritedbroot.cpp

```diff
@@ -68,6 +70,8 @@ void usage(const std::string &progn, const char *msg = 0, ...) {
           "   --search_service: Url to search service. If none is provided\n"
           "                    then uses relative url for standard Portable\n"
           "                    search.\n"
+          "   --preserve_search_service:\n" 
+          "                    Keep original search service url\n"
```

Comment:
Need a period after "url" to be consistent with the rest of the usage.

```suggestion
          "                    Keep original search service url.\n"
```